contracts/governance/Governance.sol

```diff
@@ -23,6 +22,14 @@ contract Governance is Initializable, ReentrancyGuardTransient, GovernanceSettin
         uint256[] choices; // Votes choices
     }
 
+    // ProposalState.status constants
+    uint256 constant public STATUS_INITIAL = 0;
+    uint256 constant public STATUS_RESOLVED = 1;
+    uint256 constant public STATUS_FAILED = 1 << 1;
+    uint256 constant public STATUS_CANCELED = 1 << 2;
+    uint256 constant public STATUS_EXECUTION_EXPIRED = 1 << 3;
```

Comment:
Consider introducing `enum` instead of these constants.
https://www.tutorialspoint.com/solidity/solidity_enums.htm